### ISSUE TITLE
Syntax error in src\AuthToken.php

### ISSUE THREAD

**Issue by raptor-mvk** [cloudinary/cloudinary_php#186]
Const UNSAFE contains invalid regexp
```
([ "#%&\'\/:;<=>?@\[\]^`{\|}~\\\])
```

Valid is 
```
([ "#%&\'\/:;<=>?@\[\]^`{\|}~\\])
```

**Comment by const-cloudinary** [comment]
@raptor-mvk, yes, regex is invalid, we use this regex in `preg_replace_callback` and in order to use it there `\` should be escaped twice, e.g. `\\\\`

**Comment by const-cloudinary** [comment]
Fixed in #190

**Comment by const-cloudinary** [comment]
@raptor-mvk Thank you for notifying us on that!
